test/udfs/test_abstract_udf.py

```diff
@@ -30,6 +31,9 @@ def test_udf_abstract_functions(self, mock_pipeline):
         derived_udf_classes = list(get_all_subclasses(AbstractUDF))
         # Go over each derived class of AbstractUDF
         for derived_udf_class in derived_udf_classes:
+            # skip yolo to avoid downloading model
```

Comment:
I remember facing the same issue for HF also, so I mocked the function call, maybe you can do the same? Just mock the Yolo call?

Reply:
I can't figure out mocking for yolo since it is loaded using a file path. Sticking with the current code isn't super important.